lib/capsize/template/base.rb

```diff
@@ -4,7 +4,7 @@ module Base
       CONFIG = {
         :application => 'your_app_name',
         :deploy_to => '/path/to/deployment_base',
-        :scm => 'git',
+        :scm => ':git',
```

Comment:
The `:scm` parameter has a default value that is a symbol (though it will also work with a string), but every parameter value must be a string since that is what the user will input and how the values will be processed.  This is just setting the default parameters for a new project and their default values.